src/Editor/index.js

```diff
@@ -25,8 +25,10 @@ export class Editor extends React.Component {
         let msg = ''
         let formattedMsg = ''
         if(props.initialValue && (props.initialValue !== '')){
-            msg = props.initialValue;
-            formattedMsg = this.formatTextWithMentions(props.initialValue);
+            this.buildMentionsMap.bind(this)(props.initialValue)
+            const plainText = formattedTextToPlain(props.initialValue);
```

Comment:
I would not use the extra `plainText` variable.

Reply:
Do I just call `formattedTextToPlain(props.initialValue)` twice then?

Reply:
you have it in `msg` already, so there's no need to compute it again.

Reply:
Nevermind